vg/font_test.go

```diff
@@ -0,0 +1,213 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vg
+
+import (
+	"errors"
+	"testing"
+
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/sfnt"
+	"golang.org/x/image/math/fixed"
+)
+
+func TestFontExtends(t *testing.T) {
+	for _, tc := range []struct {
+		font string
+		want map[Length]FontExtents
+	}{
+		{
+			font: "Times-Roman",
+			want: map[Length]FontExtents{
```

Comment:
How were the want values obtained?

Reply:
by extraction from what we had with `truetype`.
I'll add a comment.